static/js/containers/HomePage.js

```diff
@@ -63,6 +63,26 @@ class HomePage extends React.Component<Props> {
     dispatch(setPostData(response.posts))
   }
 
+  loadMore = async () => {
```

Comment:
It's very clear that the home page and channel page share a lot of common functionality for displaying posts, including infinite scroll. The `loadMore` logic, as well as the `<PostList>` and `<InfiniteScroll>` rendering is basically identical. I know that the channel page and home page are very likely to be the only two components that need to show posts, so if we keep this repetition I think that would be OK, but if you see a really quick, straightforward way to move this logic into an HOC or something, I would say go for it. Worth remembering that we're repeating testing logic on top of the application logic

Reply:
That's a good point, I'll look into it